__tests__/app.test.js

```diff
@@ -227,6 +230,7 @@ describe('Test POST /api/article/:article_id/comments endpoint', () => {
         })
       })
   })
```

Comment:
Please add another test with status 201 here to check if extra properties are ignored. 

For example when posting with this object:
`{username:"butter_bridge", body: "pretty cool", extraProperty: "someValue"}`